gh-pages/src/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: "default.html"
+title: "What is Baseline?"
+---
+
+Baseline gives you clear information about which web platform features work across our [core browser set](#how-do-features-become-part-of-baseline%3F) today. When reading an article, or choosing a library for your project, if the features used are all part of Baseline, you can trust the level of browser compatibility. By aligning with Baseline, there should be fewer surprises when testing your site.
+
+## How do features become part of Baseline?
+
+Baseline features are available across popular browsers. Baseline has two stages:
+
+- **Newly available**: The feature works across the latest devices and browser versions. The feature might not work in older devices or browsers. Indicated with a blue icon.
+- **Widely available**: The feature is well established and works across many devices and browser versions. It’s been available across browsers for at least 2½ years (30 months). Indicated with a green icon.
+
+Baseline is calculated using the following core browser set:
+
+- Apple Safari (macOS and iOS)
+- Google Chrome (desktop and Android)
+- Microsoft Edge (desktop)
+- Mozilla Firefox (desktop and Android)
+
+## How do I find the Baseline status of a feature?
+
+You can find at-a-glance Baseline statuses on [Can I Use](https://caniuse.com/) feature entries, [MDN Web Docs](https://developer.mozilla.org/) reference pages, and [webstatus.dev](https://webstatus.dev). See [Baseline in the wild](/baseline-in-the-wild/) for more examples.
```

Comment:
It looks like the webstatus.dev link didn't get removed from here as requested, but from a different page.

Reply:
Removed it from another page - in de35a122eee9e74ee11fd673eb1d4bb7c8321a7c, and just removed it from this page in 3eea54975d50098f4c5b140b5440a5534593f60b